data/json/items/comestibles.json

```diff
@@ -2422,12 +2422,12 @@
         "symbol" : "%",
         "quench" : -1,
         "healthy" : 1,
-        "nutrition" : 40,
+        "nutrition" : 24,
```

Comment:
I'll drop the changes, but these values are close to correct, I based them off rehydrated fish since cooked fish seems to be missing.  Salted fish is particularly broken.

Here are some valuse for comparison
1 raw fish = 32 nutrition when salted
1 raw fish = 14 nutrition when dried (16 when rehydrated)

1 raw meat = 36 nutrition when salted (jerky)
1 raw meat = 40 nutrition when dried (44 when rehydrated